arch/x86_64/src/intel64/intel64_fbterm.c

```diff
@@ -284,43 +296,45 @@ void x86_64_mb2_fbinitialize(struct multiboot_tag_framebuffer *fbt)
  * Name: fb_putc
  ****************************************************************************/
 
-void fb_putc(char ch)
+void fb_putc(int ch)
 {
   const struct nx_fontbitmap_s *fbm;
   uint8_t                       gly_x;
   uint8_t                       gly_y;
+  NXHANDLE                      font_handle = (NXHANDLE)g_fb_term.font;
 
   if (g_fb.baseaddr == NULL)
     {
-      return;
+      return ;
```

Comment:
```suggestion
      return;
```
Please remove the space thanks :-)

Reply:
Done.